pkg/noderesourcetopology/cache/passthrough.go

```diff
@@ -34,17 +36,17 @@ func NewPassthrough(lister listerv1alpha2.NodeResourceTopologyLister) Interface
 	}
 }
 
-func (pt Passthrough) GetCachedNRTCopy(nodeName string, _ *corev1.Pod) (*topologyv1alpha2.NodeResourceTopology, bool) {
+func (pt Passthrough) GetCachedNRTCopy(nodeName string, _ *corev1.Pod) (*topologyv1alpha2.NodeResourceTopology, error) {
 	klog.V(5).InfoS("Lister for nodeResTopoPlugin", "lister", pt.lister)
 	nrt, err := pt.lister.Get(nodeName)
 	if err != nil {
-		klog.V(5).ErrorS(err, "Cannot get NodeTopologies from NodeResourceTopologyLister")
-		return nil, true
+		return nil, fmt.Errorf("cannot get NodeTopologies from NodeResourceTopologyLister")
```

Comment:
this is another trap, the original code didn't fail. We need to check what happens on the call site with this change, which looks indeed more correct.

Reply:
this is what it looks like in lister:
```
	obj, exists, err := s.indexer.GetByKey(name)
	if err != nil {
		return nil, err
	}
	if !exists {
		return nil, errors.NewNotFound(v1alpha2.Resource("noderesourcetopology"), name)
	}
```
So if NRT doesn't exist we'll get Not Found error.

Reply:
Uhm, I still tend to believe we DO NOT want to error out here, I recall some past conversations in this area. I'll do dome archeology.

Reply:
let's hold on this and the other item regarding GetCachedNRTCopy. I'll do some research and come up with the suggestion. I'm confident postponing these changes should be little hassle.